snowdrop-execution/src/Snowdrop/Execution/DbActions/Types.hs

```diff
@@ -134,11 +156,48 @@ instance Buildable DbActionsException where
         DbWrongIdQuery t -> bprint ("Wrong id query to DB: "%build) t
         DbWrongPrefixIter t -> bprint ("Wrong prefix iterator: "%build) t
         DbProtocolError t -> bprint ("Db protocol error: "%build) t
-        DbUndoProjectionError -> "Wrong undo object provided: projection failed"
 
 -- | Toggle on whether to record proof within DbModifyActions
 data RememberForProof = RememberForProof { unRememberForProof :: Bool }
 
 data ClientMode proof
     = ProofMode { cmProof :: proof }
     | RemoteMode
+
+instance HFunctor (DbAccessActions conf) where
```

Comment:
This type class isn't used within this PR, but to be used later, for implementation of SD-193